library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss

```diff
@@ -464,4 +464,18 @@
 		@extend .contentReviewIconDraft;
 		color: #9E9E9E;
 	}
+
+	div#pagingHeader {
+		height: 20px;
+		padding: 15px;
+		display: flex;
+		display: -webkit-box;
+		display: -ms-flexbox;
+		display: -webkit-flex;
+		-webkit-box-align: center;
+		-webkit-align-items: center;
+		-ms-flex-align: center;
+		align-items: center;
```

Comment:
Please make use of SASS here .... you can write clean SASS and let SASS do the dirty work of making vendor-prefixed CSS:

  http://compass-style.org/reference/compass/css3/flexbox/

Reply:
No problem.